fenix/app/src/main/java/org/mozilla/fenix/onboarding/view/OnboardingPage.kt

```diff
@@ -148,7 +148,7 @@ fun OnboardingPage(
             }
 
             LaunchedEffect(pageState) {
-                pageState.onRecordImpressionEvent()
+                pageState.onRecordImpressionEvent
```

Comment:
This will cause a bug for the features which use this component to call this lambda. Let's revert this change.